README.md

```diff
@@ -19,18 +41,135 @@ features:
 
 Navigate to our [web app](https://aux.exchange) to start trading.
 
-## Using the API
+## Typescript SDK
+
+Also known as `aux-ts`.
 
 See [`aptos/api/aux-ts/examples`](./aptos/api/aux-ts/examples) for examples of interacting
 with the AMM and CLOB through typescript.
 
-## Quickstart
+### Quickstart
 
-See [Quickstart](./CONTRIBUTING.md#Quickstart) for tutorials on how to run a local instance or in a container.
+```typescript
+import { AptosClient, FaucetClient } from "aptos";
+import { AuxClient } from "../src/client";
+import { USDC_ETH_WH } from "../src/coin";
+
+const auxClient = new AuxClient(
+  "mainnet",
+  new AptosClient("https://fullnode.mainnet.aptoslabs.com")
+);
+
+const poolClient = auxClient.pool({
+  coinTypeX: "0x1::aptos_coin::AptosCoin",
+  coinTypeY: USDC_ETH_WH,
+});
+let pool = poolClient.query();
+await poolClient.swap({
+  coinTypeIn: auxCoin,
+  exactAmountIn: DU(2),
+  parameters: { slippage: new Bps(50) },
+});
+pool = poolClient.query();
+```
+
+### Walkthrough
```

Comment:
Lots of imports missing here, didn't call them all out. Generally i think any ts we put in the readme like this should run. Otherwise i'd prefer to let people just run an example. The other tough thing about this is it's hard to know when it goes out of date.